src/client.rs

```diff
@@ -41,7 +41,23 @@ impl Client {
         }
     }
 
-    /// List all [Index]es and returns values as instances of [Index].
+    fn parse_indexes_results_from_value(&self, value: Value) -> Result<IndexesResults, Error> {
+        let raw_indexes = value["results"].as_array().unwrap();
+
+        let indexes_results = IndexesResults {
+            limit: value["limit"].as_u64().unwrap() as u32,
```

Comment:
Why we don't use `as_u32()` method directly?

Reply:
The method `as_u32()` does not exists :(